docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * @file
+ * Contains hs_admin.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Markup;
+use Drupal\user\Entity\User;
+
+/**
+ * Implements hook_preprocess_node_edit_form().
+ */
+function hs_admin_page_attachments(&$variables) {
+  $logged_in = \Drupal::currentUser()->isAuthenticated();
+
+  if ($logged_in) {
+    $variables['#attached']['library'][] = 'hs_admin/hs_admin';
+  }
+}
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function hs_admin_preprocess_html(&$variables) {
+  // Get currently active user and his roles.
+  $account = \Drupal::currentUser();
+  $roles = $account->getRoles();
+
+  foreach ($roles as $role) {
+    $variables['attributes']['class'][] = 'role-' . $role;
+  }
+}
+
+/**
+ * Implements hook_toolbar_alter().
+ */
+function hs_admin_toolbar_alter(&$items) {
+  $uid = \Drupal::currentUser()->id();
+  $user = User::load($uid);
+
+  if (!$user->hasPermission('show manage link') && $uid != 1) {
+    unset($items['admin_toolbar_tools']);
+    unset($items['administration']);
+  }
+}
+
+/**
+ * Implements hook_preprocess_menu_local_tasks().
+ */
+function hs_admin_preprocess_menu_local_tasks(&$variables) {
+  // Disable edit/create node tabs for publish and delete.
+  unset($variables['primary']['entity.node.publish']);
+  unset($variables['primary']['entity.node.delete_form']);
```

Comment:
This is probably better done in a different hook besides a preprocess.

Reply:
Done, changed to `hook_menu_local_tasks_alter()`